types/office-js/index.d.ts

```diff
@@ -59500,14 +65727,14 @@ declare namespace OneNote {
      */
     function run<T>(batch: (context: OneNote.RequestContext) => Promise<T>): Promise<T>;
     /**
-     * Executes a batch script that performs actions on the OneNote object model, using the request context of a previously-created API object.
-     * @param object - A previously-created API object. The batch will use the same request context as the passed-in object, which means that any changes applied to the object will be picked up by "context.sync()".
+     * Executes a batch script that performs actions on the OneNote object model, using the request context of previously-created API objects.
```

Comment:
Are all these changes to 'OneNote.run`/`Visio.run` by design?

Reply:
Yeah, they looked to be documenting the wrong method. We also wanted the most useful signature at the end.